src/code/z_actor.c

```diff
@@ -3163,45 +3207,54 @@ void func_800328D4(PlayState* play, ActorContext* actorCtx, Player* player, u32
     }
 }
 
-u8 D_801160A0[] = {
+u8 sTargetableCategorySearchOrder[] = {
     ACTORCAT_BOSS,  ACTORCAT_ENEMY,  ACTORCAT_BG,   ACTORCAT_EXPLOSIVE, ACTORCAT_NPC,  ACTORCAT_ITEMACTION,
     ACTORCAT_CHEST, ACTORCAT_SWITCH, ACTORCAT_PROP, ACTORCAT_MISC,      ACTORCAT_DOOR, ACTORCAT_SWITCH,
 };
 
-Actor* func_80032AF0(PlayState* play, ActorContext* actorCtx, Actor** actorPtr, Player* player) {
+/**
+ * Search for the nearest targetable actor by iterating through most actor categories.
+ * See `Target_FindTargetableActorInCategory` for more details on search criteria.
+ *
+ * The actor found is stored in the targetableP parameter, which is also returned.
+ * It may be NULL if no actor that fulfills the criteria is found.
+ */
+Actor* Target_FindTargetableActor(PlayState* play, ActorContext* actorCtx, Actor** targetableP, Player* player) {
```

Comment:
I had to read this several times because using the adjective "targetable" for a noun-y thing was confusing. (On the first try I thought it was "target table" but misspelled, and on the second try I thought the pointer itself was somehow targetable, not the thing it points to.) Maybe `targetableActorP`?